src/main/java/frc/robot/shuffleboard/ShuffleboardWidgets.java

```diff
@@ -63,197 +84,233 @@
   MagazineInterface magazine;
   LiftInterface lift;
   WinchInterface winch;
-  
-
-    public ShuffleboardWidgets() {
-
-      tab = Shuffleboard.getTab("Telemetry");
-
-      /*P_testing = tab
-      .add("P", 1)
-      .withWidget(BuiltInWidgets.kNumberSlider)
-      .withSize(10, 1)
-      .withPosition(0, 1)
-      .withProperties(Map.of("min", 0, "max", 10))
-      .getEntry();
-
-      I_testing = tab
-      .add("I", 0.01)
-      .withWidget(BuiltInWidgets.kNumberSlider)
-      .withSize(10, 1)
-      .withPosition(0, 2)
-      .withProperties(Map.of("min", 0, "max", 0.1))
-      .getEntry();
-
-      D_testing = tab
-      .add("D", 10)
-      .withWidget(BuiltInWidgets.kNumberSlider)
-      .withSize(10, 1)
-      .withPosition(0, 3)
-      .withProperties(Map.of("min", 0, "max", 100))
-      .getEntry();*/
-
-      drivetrain = Robot.drivetrain;
-      turret = Robot.turret;
-      shooter = Robot.shooter;
-      magazine = Robot.magazine;
-      lift = Robot.lift;
-      winch = (WinchInterface)Robot.drivetrain;
-
-      ShuffleboardView();
 
-    }
-  
+  public ShuffleboardWidgets() {
+
+    tab = Shuffleboard.getTab("Telemetry");
+
+    /*
```

Comment:
Code Hygiene: remove commented code

Reply:
It could be of use when we do PID testing thus i kept it